packages/calcite-components/src/components/accordion/accordion.scss

```diff
@@ -1,41 +1,64 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-accordion-background-color: defines the background color for the component and it's slotted accordion-items.
+ * @prop --calcite-accordion-border-color: defines the border color for the component and it's slotted accordion-items.
+ */
+
+:host {
+  @apply relative
+    block
+    leading-6;
+  max-inline-size: var(--calcite-container-size-content-fluid);
+
+  ::slotted(accordion-item) {
+    --calcite-internal-accordion-item-icon-space-x: var(--calcite-internal-accordion-space-large);
+    --calcite-internal-accordion-item-space-inset: var(--calcite-internal-accordion-space-small)
+      var(--calcite-internal-accordion-space-large);
+  }
+}
+
+.accordion {
+  border-width: var(--calcite-border-width-sm);
+  border-color: var(--calcite-accordion-border-color, var(--calcite-color-border-2));
+  border-block-end-width: var(--calcite-border-width-none);
+  border-style: solid;
+
+  ::slotted(accordion-item) {
```

Comment:
If accordion items have these CSS props defined, shouldn't users set overrides on the items themselves since these are slotted and not owned by the accordion?

Reply:
nope! Thanks for the catch.